src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -28,6 +29,7 @@
     private static final WebDriver WEB_DRIVER = DriverManager.getInstance().getNavigator();
     private static final boolean IS_CLASSIC = PropertiesManager.getInstance().getTheme().equalsIgnoreCase("classic");
     private static final JavascriptExecutor JAVASCRIPT_EXECUTOR = (JavascriptExecutor) WEB_DRIVER;
+    private static final Logger log = Logger.getLogger(CommonActions.class.getName());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename the "log" logger to comply with the format "LOG(?:GER)?". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1312)